pkgs/development/typst-packages/cetz.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildTypstPackage,
+  fetchFromGitHub,
+  just,
+  file,
+  perl,
+}:
+
+buildTypstPackage rec {
+  pname = "cetz";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "cetz-package";
+    repo = "cetz";
+    rev = "v${version}";
+    hash = "sha256-5+Np64+0Ca8yL7D1CCUewF03Wh6BM2bV1BQNUA+bW74=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    just
+    file
+    perl
+  ];
+
+  prePatch = ''
+    patchShebangs scripts/package
+  '';
+
+  buildPhase = ''
+    export BUILD_DIR=$(mktemp -d)
+    just install $BUILD_DIR
+  '';
+
+  preInstall = ''
+    rm -rf *
+    cp -r $BUILD_DIR/cetz/0.2.2/* .
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/cetz-package/cetz";
+    description = "CeTZ: ein Typst Zeichenpaket - A library for drawing stuff with Typst.";
```

```suggestion
    description = "Library for drawing stuff with Typst";
```
Take a look at the [guidelines for `meta.description`](https://nixos.org/manual/nixpkgs/unstable/#var-meta-description).